scripts/tests/gaia_tests.bats

```diff
@@ -58,29 +58,15 @@ setup() {
 ##############################################################################################
 
 @test "[INTEGRATION-BASIC] address names are correct" {
-  assert_equal $STRIDE_ADDRESS "stride1uk4ze0x4nvh4fk0xm4jdud58eqn4yxhrt52vv7"
-  assert_equal $GAIA_ADDRESS "cosmos1pcag0cj4ttxg8l7pcg0q4ksuglswuuedcextl2"
-
-  assert_equal $DELEGATION_ICA_ADDR "cosmos1sy63lffevueudvvlvh2lf6s387xh9xq72n3fsy6n2gr5hm6u2szs2v0ujm"
-  assert_equal $REDEMPTION_ICA_ADDR "cosmos1xmcwu75s8v7s54k79390wc5gwtgkeqhvzegpj0nm2tdwacv47tmqg9ut30"
-  assert_equal $WITHDRAWAL_ICA_ADDR "cosmos1x5p8er7e2ne8l54tx33l560l8djuyapny55pksctuguzdc00dj7saqcw2l"
-  assert_equal $REVENUE_EOA_ADDR "cosmos1wdplq6qjh2xruc7qqagma9ya665q6qhcwju3ng"
-  assert_equal $FEE_ICA_ADDR "cosmos1lkgt5sfshn9shm7hd7chtytkq4mvwvswgmyl0hkacd4rmusu9wwq60cezx"
-  assert_equal $GAIA_DELEGATE_VAL_1 "cosmosvaloper1pcag0cj4ttxg8l7pcg0q4ksuglswuuedadj7ne"
-  assert_equal $GAIA_DELEGATE_VAL_2 "cosmosvaloper133lfs9gcpxqj6er3kx605e3v9lqp2pg5syhvsz"
+  assert_equal $(STRIDE_ADDRESS) "stride1uk4ze0x4nvh4fk0xm4jdud58eqn4yxhrt52vv7"
```

Comment:
just curious: why did you add the parens?

Reply:
Good question! 

So before each host address was defined as a variable like so:
```bash
STRIDE_ADDRESS=$(...)
GAIA_ADDRESS=$(...)
OSMO_ADDRESS=$(...)
etc...
```
Issue with that was when you'd source the file to load each variable, it would try to run each line, and if one of the host zones was not currently running, the script would error. So previously if, for instance, you wanted to start the network with just stride and gaia, you'd have to comment out the OSMO_ADDRESS line, otherwise the startup would fail.

I updated it so that now instead of a variable, it's a function
```
function STRIDE_ADDRESS() {...}
function GAIA_ADDRESS() {...}
function OSMO_ADDRESS() {...}
etc...
```
So now when you source the file, it simply loads the functions - but it doesn't run the code that's inside the function. And the code inside the function isn't run until you explicitly call it with $(STRIDE_ADDRESS)

Variable Access: ${VAR_NAME} or $VAR_NAME 
Function Call: $(FUNCTION_NAME)

Other benefit of this is that now there's no need to have a separate account_var script